skopt/utils.py

```diff
@@ -200,3 +210,39 @@ def func(x):
             best_fun = r.fun
 
     return [v for v in best_x], best_fun
+
+
+def cook_skopt_estimator(base_estimator, n_dims=1, is_cat=False, **kwargs):
```

Comment:
can we rename it to `make_estimator` or `make_surrogate`? `cook_` makes me think we are in Breaking Bad.